tests/numa/numa.go

```diff
@@ -107,13 +108,18 @@ var _ = Describe("[sig-compute][serial]NUMA", func() {
 })
 
 func getQEMUPID(virtClient kubecli.KubevirtClient, handlerPod *k8sv1.Pod, vmi *v1.VirtualMachineInstance) string {
-	stdout, stderr, err := tests.ExecuteCommandOnPodV2(virtClient, handlerPod, "virt-handler",
-		[]string{
-			"/bin/bash",
-			"-c",
-			"trap '' URG && ps ax",
-		})
-	Expect(err).ToNot(HaveOccurred(), stderr)
+	var stdout, stderr string
+	// The retry is a desperate try to cope with URG in case that URG is not catches by the script
+	// since URG keep ps failing
```

Comment:
is there another signal being sent or something? seems like this should work reliably.

Reply:
It is absolutely not clear to me why it is not working. With fedora bash it worked definitely. I could not make the tests pass before. With centos bash it does not seem to work but the retry seems to be good enough.